OTAnalytics/plugin_ui/customtkinter_gui/toplevel_flows.py

```diff
@@ -1,13 +1,15 @@
+import tkinter
```

Comment:
Isnt it better to `from tkinter import StringVar` instead of `import tkinter`? 

Anyway, I guess we dont necessarily need it (see comments below).